cross/arm-none-eabi-gcc/Portfile

```diff
@@ -18,6 +19,10 @@ configure.args-append \
                     --disable-newlib-supplied-syscalls \
                     --with-multilib-list=armv6-m,armv7-m,armv7e-m,cortex-m7,armv7-r
 
+# https://trac.macports.org/ticket/57153
+configure.args-append \
+                    --disable-libcc1
```

Comment:
Is this something that could be added to the PortGroup or is it too specific to just some subgroup? (I know that it doesn't affect all ports, but I'm not sure which ones are affected.)

Reply:
That is a good idea.
I moved `--disable-libcc1` into the crossgcc PG in the latest version of this pull request.
I did not add `--disable-libcc1` to `msp430-gcc` or `msp430-gcc-devel ` because I do not know if it is necessary.
Both ports are too old to build on my machine.

Reply:
msp430 ports need a serious rework, not even the homepage works any longer. The new patches seem to be at http://www.ti.com/tool/MSP430-GCC-OPENSOURCE and one can find some basic build instructions at https://gist.github.com/0/631c9006a672458eaa48.

(I would probably not bother adding any patches whatsoever as long as it's broken and useless anyway.)

@g5pw: any chance to bribe you to fix these ports?